### PR TITLE
ingress: Set interval annotation if omitted

### PR DESCRIPTION
**Pull request by rxbn**

### CODE REVIEW

File: docs/content/core_concept.md [hunk 1]
```diff
@@ -113,85 +113,85 @@ spec:
...
+   kind: IP
+   metadata:
+     name: external-ipv4
+   spec:
+     type: dynamic
+     dynamicIPSources:
+       - https://api.ipify.org
+     interval: 5m
+   ```
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 2]
```diff
@@ -113,85 +113,85 @@ spec:
...
+   spec:
+     type: dynamic
+     dynamicIPSources:
+       - https://api.ipify.org
+     interval: 5m
+   ```
+   
+   !!! warning
+       The source must return only the external IPv4 address.
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 3]
```diff
@@ -113,85 +113,85 @@ spec:
...
+     dynamicIPSources:
+       - https://api.ipify.org
+     interval: 5m
+   ```
+   
+   !!! warning
+       The source must return only the external IPv4 address.
+   
+       Good example:
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 4]
```diff
@@ -113,85 +113,85 @@ spec:
...
+   
+   !!! warning
+       The source must return only the external IPv4 address.
+   
+       Good example:
+   
+       ```bash
+       curl https://api.ipify.org
+       ```
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 5]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       The source must return only the external IPv4 address.
+   
+       Good example:
+   
+       ```bash
+       curl https://api.ipify.org
+       ```
+   
+       Output:
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 6]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       ```bash
+       curl https://api.ipify.org
+       ```
+   
+       Output:
+   
+       ```console
+       142.251.36.35
+       ```
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 7]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       ```
+   
+       Output:
+   
+       ```console
+       142.251.36.35
+       ```
+   
+       Bad example:
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 8]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       ```console
+       142.251.36.35
+       ```
+   
+       Bad example:
+   
+       ```bash
+       curl "https://api.ipify.org?format=json"
+       ```
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 9]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       ```
+   
+       Bad example:
+   
+       ```bash
+       curl "https://api.ipify.org?format=json"
+       ```
+   
+       Output:
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

File: docs/content/core_concept.md [hunk 10]
```diff
@@ -113,85 +113,85 @@ spec:
...
+       ```bash
+       curl "https://api.ipify.org?format=json"
+       ```
+   
+       Output:
+   
+       ```console
+       {"ip":"142.251.36.35"}
+       ```
+
```
Comment (gi8lino) [inline docs/content/core_concept.md]:
```suggestion

```

**Review by gi8lino — CHANGES_REQUESTED** [review]
see suggestion, rest LGTM

File: docs/content/core_concept.md [hunk 11]
```diff
@@ -201,16 +201,19 @@ To skip the creation of a `DNSRecord`, add the annotation `cf.containeroo.ch/ign
...
+| `cf.containeroo.ch/interval` | `5m`                   | Interval at which cloudflare-operator will compare the Cloudflare DNS record with the `DNSRecord` object and reconcile the DNSRecord object |
+| `cf.containeroo.ch/ignore`   | `true` or `false`      | Skip creation of a DNS record                                                                                                               |
 
 !!! note "cf.containeroo.ch/ignore"
-    If you add the label `cf.containeroo.ch/ignore=true` and cloudflare-operator has already created a `DNSRecord`, cloudflare-operator will clean up the `DNSRecord` (Kubernetes and Cloudflare)!
+    If you add the label `cf.containeroo.ch/ignore=true` and cloudflare-operator has already created a `DNSRecord`, cloudflare-operator will clean up the `DNSRecord` (Kubernetes and Cloudflare).
+
+!!! note "cf.containeroo.ch/interval"
+    If you delete the `DNSRecord` object, cloudflare-operator will automatically recreate it with the `interval` specified in the annotation.
```
Comment (rxbn) [inline docs/content/core_concept.md]:
```suggestion
    If you delete the `DNSRecord` object, cloudflare-operator will automatically recreate at the next `Ingress` reconciliation loop. If not specified, the default interval `5m` will be used.
```

**Review by gi8lino — APPROVED** [review]
LGTM

**Pull request merged**